src/commands/dispatchers/sync.ts

```diff
@@ -11,8 +11,13 @@ export const builder = (yargs: Argv) => epilogue(yargs).options({
     describe: 'Path to the file containing the Dispatcher(s) configuration',
     type: 'string',
   },
+  clean: {
+    demandOption: false,
+    describe: 'Delete Dispatchers created using the CLI, that are no longer present in the local Dispatcher file',
+    type: 'boolean',
+  },
```

Comment:
Is this false by default? Do we want this true by default? just thinking

Reply:
Does it need to be added to the global sync as well?

Reply:
It should always default to false as it's destructive yeah, and you're absolutely right about adding it to the global sync great catch!